src/node/rpc/endpoint_registry.h

```diff
@@ -269,6 +269,7 @@ namespace ccf
       Endpoint& set_allowed_verb(RESTVerb v)
       {
         const auto previous_verb = verb;
+        verb = v;
```

Comment:
I think this is the only functional change (for a deprecated feature). @eddyashton can you confirm if this is correct?